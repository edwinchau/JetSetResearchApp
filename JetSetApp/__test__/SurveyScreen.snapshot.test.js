import React from "react";
import SurveyScreen from "../app/screens/SurveyScreen"
import renderer from "react-test-renderer"

describe("<SurveyScreen/>", ()=>{
  it("SurveyScreen snapshot", ()=>{
    let wrapper = null;
    const spyNavigate = jest.fn();
    const props = {
      navaigation: {
        navigate: spyNavigate,
        state:{}
      }
    }
    const params = {
      token: 'someToken'
    }

    beforeEach(() => {
      wrapper = renderer.create(<SurveyScreen {...props} />)
      wrapper.setState({params: params})
    })
    expect(wrapper).toMatchSnapshot();
  });
});
