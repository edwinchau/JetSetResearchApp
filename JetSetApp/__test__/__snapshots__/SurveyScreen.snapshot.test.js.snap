// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SurveyScreen/> SurveyScreen snapshot 1`] = `null`;
